doc/big-picture.md

```diff
@@ -0,0 +1,67 @@
+IPLD Big Picture
+================
+
+IPLD is a system for describing, storing, sharing, and hashing data.
+
+
+Layers
+------
+
+IPLD comes in several layers:
+
+1. The IPLD Data Model
+2. The IPLD Schema system
+3. "IPFN"
+
+The IPLD Data Model is a specification of the core serializable types of data.
+You can think of it as roughly isomorphic to the familiar JSON and CBOR
+serialization systems (and yes, those parsers usually work "out of the box").
+You can read more about
+[the Data Model specification in the Specs repo](https://github.com/ipld/specs/blob/master/IPLD-Data-Model-v1.md).
+
+The IPLD Schema system builds atop the IPLD Data Model to add a system of
+user-defined types.  These type schema are meant to be easy to write, easy to
+read, and easy for the machine to parse and check efficiently.
+IPLD Schema bring common concepts like "structs" with named "fields", as well
+as "enums" and even "unions" -- and make clear definitions of how to map all of
+these concepts onto the basic serializable types of the Data Model.
+
+You can use IPLD at any of these levels you choose.  We recommend having a
+schema for your data, because it makes validity checking, sanitization,
+migration, and documentation all easier in the long run -- but the Data Model
+is also absolutely usable stand-alone.
+
+IPFN is mentioned here as an intent to design something between "dependent types"
```

Comment:
I think this is the only mention of this here (aside from the list at the top), is there a link or an expansion on the acronym you can provide to give some clarity? It's not clear to me what "not to expect" from these docs from this paragraph alone.

Reply:
Um, I see your point and will take suggestions for wording.

"IPFN" isn't a... thing that... exists.  There's been some floating discussion of "what if we made a functional programming language" and "what if WASM" and sometimes it floats into territory I'd file as "what if dependent types".  I wanted to give a tag to all that mostly to say "this ain't it".

It's possible this just isn't a good place in the docs for that.  Maybe the whole '3.' should be dropped and s/several/two/ ?

Reply:
There's some mention of non-turing-completeness and not-dependent-types in the schema page already, so that would also make this seem more droppable.

Reply:
yeah, I'd say drop it, it's quite a tangent and maybe deserves a more complete write-up of the ideas in a different place

Reply:
sold :+1: